felt/src/bigint_felt.rs

```diff
@@ -170,13 +170,22 @@ impl FeltOps for FeltBigInt<FIELD_HIGH, FIELD_LOW> {
     }
 
     fn from_bytes_be(bytes: &[u8]) -> FeltBigInt<FIELD_HIGH, FIELD_LOW> {
-        let mut value = BigUint::from_bytes_be(bytes);
-        if value >= *CAIRO_PRIME_BIGUINT {
-            value = value.mod_floor(&CAIRO_PRIME_BIGUINT);
-        }
+        let value = BigUint::from_bytes_be(bytes);
         Self::from(value)
     }
 
+    fn from_bytes_le(bytes: &[u8]) -> FeltBigInt<FIELD_HIGH, FIELD_LOW> {
+        Self::from(BigUint::from_bytes_le(bytes))
+    }
+
+    fn from_bytes_ne(bytes: &[u8]) -> FeltBigInt<FIELD_HIGH, FIELD_LOW> {
+        #[cfg(target_endian = "little")]
```

Comment:
Sorry @MegaRedHand 
I don't get want this line does, can you explain it a little bit?

Reply:
Sure! It enables the code block after it, only if the compilation target is _little-endian_. The same goes for the next part, although for _big-endian_ targets.